### PR TITLE
Add plausible

### PR DESCRIPTION
**Pull request by adpi2**
It only activates plausible on production. But after this will be merged to develop we will be able to use the `Env` object to activate plausible to pre-prod too.

### CODE REVIEW

File: server/src/main/scala/ch.epfl.scala.index.server/Server.scala [hunk 1]
```diff
@@ -49,12 +49,13 @@ object Server {
     // because of the sbtResolver mode
     val data = DataRepository.open()
 
-    val searchPages = new SearchPages(data, session)
+    val searchPages = new SearchPages(config.production, data, session)
```
Comment (mlachkar) [inline server/src/main/scala/ch.epfl.scala.index.server/Server.scala:52]: I was about to comment.. what is this config ? it should be config.env ^^

**Review by mlachkar — APPROVED** [review]
I don't get why we would need analytics on preprod! Preprod is only for development purposes, so we don't really expect users on it ?

File: server/src/main/scala/ch.epfl.scala.index.server/Server.scala (hunk 1 above)
Comment (adpi2) [reply server/src/main/scala/ch.epfl.scala.index.server/Server.scala:52]: ^^

**Comment by adpi2** [comment]
I think we need it on preprod just to test that it works.

**Pull request merged**